build/schema.js

```diff
@@ -13,21 +13,27 @@
             "properties": {
                 "alphabet": {
                     "type": "string",
-                    "minlength": 4,
-                    "maxlength": 4
+                    "pattern": "^[A-Z]{1}[a-z]{3}$"
                 },
                 "continent": {
-                    "type": "string",
-                    "minimum": 4,
-                    "maximum": 4
+                    "anyOf": [{
+                        "type": "string",
+                        "pattern": "^[A-Z]{2}$"
+                    }, {
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "pattern": "^[A-Z]{2}$"
+                        }
+                    }]
                 },
                 "language": {
                     "type": "string",
-                    "minlength": 1
+                    "pattern": "^[A-Za-z]+$"
```

Comment:
Since markdown renders carriage returns, should we remove the hard returns that wrap the text around 80 characters, or include them?